### PR TITLE
Don't break when a calendar is missing

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: auto_nag/round_robin.py [hunk 1]
```diff
@@ -67,16 +67,25 @@ def feed(self, teams, rr=None):
         # - components -> dictionary: Product::Component -> strategy name
         # - strategies: dictionary: {calendar: url}
 
+        torm = []
```
Comment (marco-c) [inline auto_nag/round_robin.py:70]: torm?

File: auto_nag/round_robin.py [hunk 2]
```diff
@@ -67,16 +67,25 @@ def feed(self, teams, rr=None):
...
+                calendars = self.get_calendar(team, data)
+                self.all_calendars += list(calendars.values())
+
+                # finally self.data is a dictionary:
+                # - Product::Component -> dictionary {fallback: who to nag when we've nobody
+                #                                     calendar}
+                for pc, strategy in data["components"].items():
+                    self.data[pc] = calendars[strategy]
+            except (BadFallback, InvalidCalendar) as err:
+                logger.error(err)
```
Comment (marco-c) [inline auto_nag/round_robin.py:84]: Are these errors reported to us via email?

File: auto_nag/round_robin.py (hunk 1 above)
Comment (calixteman) [reply auto_nag/round_robin.py:70]: to remove

File: auto_nag/round_robin.py (hunk 2 above)
Comment (calixteman) [reply auto_nag/round_robin.py:84]: Normally yes

**Review by marco-c — APPROVED** [review]

**Pull request merged**